gzhttp/compress.go

```diff
@@ -236,7 +235,7 @@ func (w *GzipResponseWriter) startGzip(remain []byte) error {
 				}
 				jitRNG = binary.LittleEndian.Uint32(tmp[:])
 			}
-			jit := string(w.randomJitter[:1+jitRNG%uint32(len(w.randomJitter)-1)])
+			jit := w.randomJitter[:1+jitRNG%uint32(len(w.randomJitter)-1)]
 			//fmt.Println("w.buf:", len(w.buf), "remain:", len(remain), "jitter:", len(jit))
 			w.gw.(writer.GzipWriterExt).SetHeader(writer.Header{Comment: &jit})
```

Comment:
This line still has to copy the string header for jit to the heap. Would it be acceptable for SetHeader to set the Comment field unconditionally? Then it could just be a string instead of a *string.